chopper/lib/src/response.dart

```diff
@@ -1,3 +1,4 @@
+import 'dart:io';
```

Comment:
This excludes browser based Chopper apps since `dart:io` is only available for:

- Servers
- Command-line scripts
- Flutter mobile apps
- Flutter desktop apps

https://api.dart.dev/stable/3.2.4/dart-io/dart-io-library.html

Reply:
Forgot it.. added custom exception instead

Reply:
The import still exists though. 😅

Reply:
Seriously 🤦  OMG...